modules/utils/docker/bridge_settings.sh

```diff
@@ -50,20 +50,55 @@ if [ -f "$COMMS_PCB_VERSION_FILE" ]; then
 fi
 echo $eth_port
 
-# Set mtu back to 1500 to support e2e connectivity
-# TODO: Investigate this as it should still work with 1460
-ifconfig bat0 mtu 1500
-ifconfig br-lan mtu 1500
 
-brctl addif br-lan bat0 $eth_port
-echo $br_lan_ip
-ifconfig br-lan $br_lan_ip netmask "255.255.255.0"
-ifconfig br-lan up
-echo
-ifconfig br-lan
-# Add forwarding rules from AP to $mesh_if interface
-iptables -P FORWARD ACCEPT
-route del -net 192.168.1.0 gw 0.0.0.0 netmask 255.255.255.0 dev br-lan
-route add -net 192.168.1.0 gw $br_lan_ip netmask 255.255.255.0 dev br-lan
-iptables -A FORWARD --in-interface bat0 -j ACCEPT
-iptables --table nat -A POSTROUTING --out-interface $br_lan_ip -j MASQUERADE
\ No newline at end of file
+
+
+# Bridge ethernet and Mesh for Batman-avd and routing for QoS-OLSR
+if [ "$algo" = "olsr" ]; then
+  brctl addif br-lan "$eth_port"
+  IPV6_PREFIX=$( echo fd`dd if=/dev/urandom bs=7 count=1 status=none | xxd -p` | sed 's/\(....\)/\1:/g' )
```

Comment:
Not using SLAAC?

Reply:
I think @TIISR also had questions about this - for now let's leave it and we are working on another PR to change everything to SLAAC